core/indexer/elasticsearchDatabase.go

```diff
@@ -165,12 +173,11 @@ func (esd *elasticSearchDatabase) getSerializedElasticBlockAndHeaderHash(
 		Proposer:              signersIndexes[0],
 		Validators:            signersIndexes,
 		PubKeyBitmap:          hex.EncodeToString(header.GetPubKeysBitmap()),
-		// TODO compute correctly size of block = size(header) + size(body) + size(transactions)
-		Size:          int64(len(h)),
-		Timestamp:     time.Duration(header.GetTimeStamp()),
-		TxCount:       header.GetTxCount(),
-		StateRootHash: hex.EncodeToString(header.GetRootHash()),
-		PrevHash:      hex.EncodeToString(header.GetPrevHash()),
+		Size:                  int64(blockSizeInBytes),
```

Comment:
Maybe add the field `HeaderSize: len(headerBytes) + len(bodyBytes)` ?

Reply:
done.